File: taloa-web/types/vet.ts
export interface PublicVet {
  id: string;
  name: string;
  phone: string;
  address: string | null;
  area: string | null;
  species_supported: string[] | null;
  emergency_24h: boolean;
  hours: string | null;
  website: string | null;
}

File: taloa-api/app/schemas/vet.py
"""Schema publico de clinica veterinaria (Emergency Directory)."""
from pydantic import BaseModel


class PublicVet(BaseModel):
    id: str
    name: str
    phone: str
    address: str | None = None
    area: str | None = None
    species_supported: list[str] | None = None
    emergency_24h: bool = False
    hours: str | None = None
    website: str | None = None
